### DISCUSSION TITLE
Tutorials

### DISCUSSION THREAD

**Discussion by hgarner44** [goma/goma#445] [General]
Are the Tutorials listed at the end of the User Manual publicly available somewhere?

Thanks

**Comment by wortiz** [comment]
They've only been available internally at Sandia / UNM / Industry partners until recently. I believe we do have the capability to share some of them. 

Some tutorials have not been updated in a while and we want to make an effort to make sure they are correct before we release publicly.

If you are looking for a specific tutorial you can ask for more information by email: https://www.gomafem.com/contact.html

If you are trying to see if Goma can do a specific problem in your interest you could let us know more information here or by email and we can give tips or relevant tutorials / sample problems or let you know if it isn't in Goma's current wheelhouse.
